core/controllers/editor.py

```diff
@@ -790,6 +1199,39 @@ class LearnerAnswerInfoHandler(EditorHandler):
     """Handles the learner answer info for an exploration state."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'entity_type': {
+            'schema': {
+                'type': 'basestring',
+                'choices': [
+                    feconf.ENTITY_TYPE_EXPLORATION,
+                    feconf.ENTITY_TYPE_QUESTION
+                ]
+            }
+        },
+        'entity_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'DELETE': {
+            'state_name': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
Please remove the None and the exception.

Reply:
Done

Reply:
I'm not sure if this is right, but it might be because this handler should be split into two, one for exps and one for questions, but I'm not sure about that.